src/btree/bt_prefetch.c

```diff
@@ -80,6 +76,11 @@ __wt_btree_prefetch(WT_SESSION_IMPL *session, WT_REF *ref)
         if (next_ref->state == WT_REF_DISK && F_ISSET(next_ref, WT_REF_FLAG_LEAF) &&
           next_ref->page_del == NULL && !F_ISSET(next_ref, WT_REF_FLAG_PREFETCH)) {
             ret = __wt_conn_prefetch_queue_push(session, next_ref);
+            /* Increment concurrent eviction block stat here? */
```

Comment:
This was a suggestion, we don't need a stat but maybe one day it will come in useful?

Reply:
I think we are okay to leave this as is for now until we find a need in the future.